BuildForPublication.cmd

```diff
@@ -3,12 +3,21 @@ goto :main
 
 :main
 setlocal
+  set errorlevel=
+  set BuildConfiguration=Release
+  set VersionSuffix=build0047
+  set PackageVersion=1.0.0-alpha-%VersionSuffix%
 
-set PackageVersion=1.0.0-alpha-build0043
-set VersionSuffix=build0043
+  echo/==================
+  echo/ Building version %PackageVersion% NuGet packages.
+  echo/==================
 
-echo Building version %PackageVersion% NuGet packages.
+  set LocalDotNet_ToolsDir=%~dp0tools
+  if exist "%LocalDotNet_ToolsDir%" rmdir /s /q "%LocalDotNet_ToolsDir%"
```

Comment:
@brianrob This script is build for publication, and the idea is to have a clean repo when you want to publish a new package.

---
In reply to: [100207679](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100207679) [](ancestors = 100207679)